travis-run-tests.sh

```diff
@@ -1,8 +1,10 @@
 #! /usr/bin/env bash
 
+set -e
+
 current_version=`python setup.py --version`
 
-if ! git diff-index --quiet master openfisca_web_api
+if [[ $(git --no-pager diff --ignore-blank-lines  master openfisca_web_api) ]]
```

Comment:
Do you see an alternative solution to return an `error` code only if there is a diff inside the `openfisca_web_api` directory, **and** if this diff is not just a blank line ?